deepseg.cc

```diff
@@ -227,11 +230,12 @@ int main(int argc, char* argv[]) {
 	if (showUsage) {
 		fprintf(stderr, "\n");
 		fprintf(stderr, "usage:\n");
-		fprintf(stderr, "  deepseg [-?] [-d] [-c <capture>] [-v <virtual>] [-w <width>] [-h <height>]\n");
+		fprintf(stderr, "  deepseg [-?] [-d] [-p] [-c <capture>] [-v <virtual>] [-w <width> -h <height>]\n");
```

Comment:
While it doesn't usually make much sense to leave out width or height, but specify the other, it's not forbidden by program logic to do so. And as they have default values just providing one may work just as intended (`-w 800` to get `800x480` as resolution or single `-h 400` to get `640x400` which are both valid resolutions).

Also this is an unrelated change that's better suited for another PR …

Reply:
Totally agree, I will revert the changes around width and height. Thanks for reviewing!